### ISSUE TITLE
Requires manual install of several packages

### ISSUE THREAD

**Issue by jpotts** [Alfresco/alfresco-js-api#52]
If you just do an npm install --save alfresco-api-js and then try to run the following test JS file:

```
var AlfrescoApi = require('alfresco-js-api');

this.alfrescoJsApi = new AlfrescoApi();

this.alfrescoJsApi.login('admin', 'admin').then(function (data) {
    console.log('API called successfully Login ticket:' + data);
}, function (error) {
    console.error(error);
});
```

You will get an exception that "event-emitter" cannot be found.

If you then do an npm install --save event-emitter you will get "lodash" cannot be found.

If you then do an npm install --save lodash you will get "nock" cannot be found.

Finally, doing an npm install --save nock resolves the issue.

**Comment by magemello** [comment]
Hi Jeff,

We published the 0.3.2 version, where this problem should be fixed. Can you confirm it?

**Comment by jpotts** [comment]
Confirmed, thanks for fixing!
